tests/phpunit/TemplateTest.php

```diff
@@ -182,6 +182,10 @@ public function testMisspeltParameters() {
     $this->assertNotNull($expanded->get('journal'));
     $this->assertNotNull($expanded->get('pages'));
     $this->assertNotNull($expanded->get('year'));
+      
+    $text = "{{Cite journal|pp. 1–5}}";
+    $expanded = $this->process_citation($text);
+    $this->assertNotNull($expanded->get('pages'));
```

Comment:
Is it worth using `AssertEquals('1–5', $expanded->get('pages');`?

Reply:
Will do